### DISCUSSION TITLE
CCSID

### DISCUSSION THREAD

**Discussion by basim1971** [codefori/vscode-ibmi#2113] [General]
Hi all,
I have the following definition
- usrprf  ccsid 280 (Italy)
- srcfile ccsid 280 (italy)
- connection on code for ibm with CCSID *FILE 
- QCCSID system value 37
When i open a source file with special char like § it's convert to @ . It's seem that visual code  use the QCCSID system value

Can someone explain to me how  CCSID are manage ?

Thanks

Basilio

**Comment by chrjorgensen** [comment]
@basim1971 Please try the prerelease version 2.10.7, which solves a bug concerning the CCSID setting - and use `Connect and Reload Server Settings` - and see if the problem is fixed.

**Comment by basim1971** [comment]
@chrjorgensen  unfortunately it does not work

**Comment by worksofliam** [comment]
@basim1971 Are you able to share the actual error or issue you're getting?

It seems we have fixed this before: https://github.com/codefori/vscode-ibmi/issues/719
